### ISSUE TITLE
KD Training for Cross Encoder

### ISSUE THREAD

**Issue by aqx95** [UKPLab/sentence-transformers#2290]
Thank you very much for this wonderful repo!
I would like to refer to the training script for knowledge distillation on cross encoder. https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/ms_marco/train_cross-encoder_kd.py
From the description, it mentioned that the distillation process is based on Hofstatter et al setup. According to the paper, they use a Margin MSE loss for the procedure, while the script uses a MSE Loss. 
May i know if there is any reason for using MSE instead of Margin MSE.
Thank you.
